actgen/train.py

```diff
@@ -41,6 +41,9 @@ def parse_args(self):
         parser.add_argument('--agent', type=str, default='dqn',
                             choices=['dqn', 'random', 'action_dqn'],
                             help='Which agent to use')
+        parser.add_argument('--regularization', type=str, default='None',
+                            choices=['None', 'l1', 'l2', 'l1_l2', 'dropout', 'l1_dropout', 'l2_dropout', 'l1_l2_dropout'],
```

Comment:
What is `l1_l2`?

Reply:
I don't think we need to investigate the combinations of all the different regularizations (e.g. `l1_dropout`)

Reply:
`l1_l2` is using l1 and l2 regularization at the same time. I'm not sure if people actually use this.

Reply:
I haven't heard of people doing that. People do use dropout and other regularization concurrently, but the point of the experiment is to see if any of the obvious types of regularization confers a generalization benefit.